parmed/amber/_amberparm.py

```diff
@@ -2237,6 +2563,60 @@ def __setstate__(self, d):
                 setattr(self, attr, d[attr])
 
 
+# ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+class CMAP(object):
```

Comment:
What is this object used for? There's already a CMAP object in `topologyobjects`.